src/distributed_ls/Euclid/mat_dh_private.c

```diff
@@ -356,7 +358,10 @@ if (feof(fp)) hypre_printf("trouble!");
   /* reset file, and skip over header again */
   rewind(fp);
   for (i=0; i<ignore; ++i) {
-    fgets(junk, MAX_JUNK, fp);
+    if (fgets(junk, MAX_JUNK, fp) == NULL) {
```

Comment:
Need change the code style

```
if (...)
{
    ...
}
```

Reply:
Hi Ruipeng, I followed the code style used by Euclid. Rob told it was fine. I guess Euclid was not updated to the style we use nowadays. By the way, do you have some sort of config file to ensure automatically a code style when opening source files with a specific editor?

Reply:
It is a minor issue. Never mind if Euclid used this style. We had discussions on automatically change code style before. It is fine to leave as is for now.